example.config.yml

```diff
@@ -12,13 +12,16 @@ resolution: { }
 total_images: 10
 
 # The interval between captures (in seconds).
-interval: 1
+interval: 5
 
 # Whether to create an animated gif after capturing the time-lapse images.
-create_gif: False
+create_gif: True
```

Comment:
I'd rather keep this default (and `create_video`) `False` for now, just because on any of the longer timelapses I've done, I found it took many minutes, sometimes _hours_, to create the gif or video on the Pi. So instead, I shut down the Pi, put the card into my desktop, then let the desktop crunch all the pictures together... usually about 100x faster :)